bravado/response.py

```diff
@@ -29,10 +29,18 @@ class BravadoResponseMetadata(object):
     use at your own risk.
     """
 
-    def __init__(self, incoming_response, swagger_result, elapsed_time, exc_info):
+    def __init__(self, incoming_response, swagger_result, elapsed_time, handled_exception_info):
+        """
+        :param incoming_response: a subclass of bravado_core.response.IncomingResponse.
+        :param swagger_result: the unmarshalled result that is being returned to the user.
+        :param elapsed_time: float containing an approximate elapsed time since creating the future, in seconds.
+        :param handled_exception_info: sys.exc_info() data if an exception was caught and handled as
+            part of a fallback response; note that the third element in the list is a string representation
```

Comment:
I would say that the third element should contain the string representation of the traceback and not the traceback due to circular reference.

It feels strange to me saying that the third element is the string representation as it the caller to decide what it is

Reply:
Heh, but the caller will be bravado code. I see this more as a documentation for a) people wanting to use the metadata and b) people wanting to implement their own version of the class. So we know what the argument will be. :)